src/main/java/org/folio/repository/UserTenantRepository.java

```diff
@@ -111,6 +113,8 @@ private UserTenant mapRowToUserTenant(Row row) {
       .withEmail(row.getString(EMAIL))
       .withPhoneNumber(row.getString(PHONE_NUMBER))
       .withMobilePhoneNumber(row.getString(MOBILE_PHONE_NUMBER))
-      .withCentralTenantId(row.getString(CENTRAL_TENANT_ID));
+      .withCentralTenantId(row.getString(CENTRAL_TENANT_ID))
+      .withBarcode(row.getString(BARCODE))
+      .withExternalSystemId(row.getString(EXTERNAL_SYSTEM_ID));
```

Comment:
Need to add/adjust existing unit tests with new logic

Reply:
Adjusted existing unit test.